### PR TITLE
Move EndShipper to general availability

### PR DESCRIPTION
**Pull request by nwithan8**
# Description

- Copy EndShipper to main namespace
- Mark beta EndShipper as deprecated
- Update unit tests

# Testing

- Unit tests for EndShipper updated and passing

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: lib/EasyPost/EndShipper.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+ * @property string $id
+ * @property string $street1
+ * @property string $street2
+ * @property string $city
+ * @property string $state
+ * @property string $zip
+ * @property string $country
+ * @property string $name
+ * @property string $company
+ * @property string $phone
```
Comment (Justintime50) [inline lib/EasyPost/EndShipper.php:16]: Issue: Email is missing here and I'd probably put name and company before street1 in the logical ordering.

**Review by Justintime50 — CHANGES_REQUESTED** [review]
We need to update the `require_once` line in the `/lib/easypost.php` file for users that manually require our library, I'm not seeing that here.

**Comment by nwithan8** [comment]
> We need to update the `require_once` line in the `/lib/easypost.php` file for users that manually require our library, I'm not seeing that here.

Thanks, I knew there was something along those lines, couldn't remember what.

File: lib/EasyPost/EndShipper.php (hunk 1 above)
Comment (nwithan8) [reply lib/EasyPost/EndShipper.php:16]: Resolved.

**Review by Justintime50 — APPROVED** [review]

**Comment by Justintime50** [comment]
> Thanks, I knew there was something along those lines, couldn't remember what.

May be good to ask next time then 😅

**Review by jchen293 — APPROVED** [review]

**Pull request merged**